TowerForge/TowerForge/LevelManager/Components/PlayerComponent.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  PlayerComponent.swift
+//  TowerForge
+//
+//  Created by Vanessa Mae on 17/03/24.
+//
+
+import Foundation
+import SpriteKit
+
+enum Player: Int {
+    case ownPlayer = 1
+    case oppositePlayer = 2
+    
+    func getOppositePlayer() -> Player {
+        switch self {
+        case .ownPlayer:
+            return .oppositePlayer
+        case .oppositePlayer:
+            return .ownPlayer
+        }
+    }
+    
+    func getDirectionVelocity() -> CGVector {
+        switch self {
+        case .ownPlayer:
+            return CGVector(dx: 1.0, dy: 0.0)
+        case .oppositePlayer:
+            return CGVector(dx: -1.0, dy: 0.0)
+        }
+    }
+}
+
+class PlayerComponent: TFComponent {
+    public var player: Player
+    public var lifeLeft: Int
+    public var points = 0
+    private var lastPointIncrease = TimeInterval(0)
+    private var pointInterval: TimeInterval
+    private var pointsPerInterval: Int = 10
```

Comment:
```suggestion
    private var pointsPerInterval: Int = 10
    private var timeSinceLastPointIncrease = TimeInterval(0)
```

This is for the comment below.